specification/monitor/resource-manager/readme.md

```diff
@@ -507,6 +507,7 @@ These settings apply only when `--tag=package-2015-07-01-only` is specified on t
 ```yaml $(tag) == 'package-2015-07-01-only'
 input-file:
 - Microsoft.Insights/stable/2015-07-01/serviceDiagnosticsSettings_API.json
+- Microsoft.Insights/stable/2014-04-01/alertRules_API.json
```

Comment:
Why you add 2014-04-01 file in tag 2015-07-01-only? The tag with -only is not support multi version.

Reply:
Our API had the lowest version of 2016-03-01, and this is a place that is prior to that date. Therfore, is seems like for the version "2015-07-01-only" our API does not exist. Please correct me if I'm wrong, I will remove that line